src/Components/Aspire.Azure.AI.OpenAI/AssemblyInfo.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Azure.AI.OpenAI;
+using Aspire;
+using Azure.AI.OpenAI;
+
+[assembly: ConfigurationSchema("Aspire:Azure:AI:OpenAI", typeof(AzureOpenAISettings))]
+[assembly: ConfigurationSchema("Aspire:Azure:AI:OpenAI:ClientOptions", typeof(OpenAIClientOptions), exclusionPaths: ["Default"])]
+
+[assembly: LoggingCategories(
+    "Azure",
+    "Azure.AI.OpenAI",
```

Comment:
Do you know who/where logs in this category? I am surprised they have a special one.